src/it/resources/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/InputNoFinalizer.java

```diff
@@ -1,27 +1,25 @@
 package com.google.checkstyle.test.chapter6programpractice.rule64finalizers;
 
-public class InputNoFinalizer
-{
-    public void finalize() // violation 'Avoid using finalizer method.'
-    {
-        // It's not enough to check if the METHOD_DEF branch contains a PARAMETER_DEF, as that would
-        // treat this method as having a parameter.
-        Runnable runnable = new Runnable() {
+/** Some javadoc. */
+public class InputNoFinalizer {
```

Comment:
@Zopsss , please try to not add javadocs, and simply format.
I plan to persuade mentors to allow single review merge on just formatting.

Reply:
Eventually , yrs, but please do not complicate formatting updates. We should merge all formatting quickly.

Reply:
got it, will keep this in mind